include/ninshiki_cpp/node/ninshiki_cpp_node.hpp

```diff
@@ -49,15 +49,10 @@ class NinshikiCppNode
 
   NinshikiCppNode(
     rclcpp::Node::SharedPtr node, const std::string & path,
-    int frequency, shisen_cpp::Options options,
-    std::shared_ptr<DnnDetector> dnn_detection,
+    int frequency, std::shared_ptr<DnnDetector> dnn_detection,
```

Comment:
It's actually not related with what you're trying to change, but it's better to pass an `std::shared_ptr` as a constant reference.